testdata/serviceconfig.yml

```diff
@@ -30,19 +30,10 @@ rules:
           response_code: response.code | 200
   - kind: access-logs
     params:
-      logName: "access_log"
+      logName: access_log
       log:
-        logFormat: 1 # COMMON
-        templateExpressions:
```

Comment:
Oh and make sure you're looking at the latest pushed commit; if you click on the file name here it goes to the commit that you commented on, not the latest thing I pushed (tricked me at first too).